### ISSUE TITLE
Hyrule Warriors Broken SFX Audio

### ISSUE THREAD

**Issue by EatYOuRNuGGeT** [cemu-project/Cemu#1086]
### Current Behavior

In Hyrule Warriors all of the SFX sounds have really bad  audio quality compared to on console and there is constant static/crackling sounds when certain SFX plays for example when throwing bombs when a boss stomps on the ground etc. 
The music in the game sounds completely fine and the audio issue is just related to the SFX. None of the audio settings in cemu can remedy this issue and all of the footage i've watched on Youtube of people playing the game on Cemu have this problem.

### Expected Behavior

For the SFX audio to work properly. 

### Steps to Reproduce

Just boot up the game and listen to the audio, you'll notice a reduction in audio quality and it basically sounds as if your speakers are blown.  

### System Info (Optional)

OS: Windows 10
GPU: RTX 3070

**Comment by Squall-Leonhart** [comment]
Log should really not be optional, but here we are, log please.

**Comment by Squall-Leonhart** [comment]
Yeah, thought so - This title makes use of Sound features not presently HLE emulated and need the CafeLibs snd_user.rpl and snduser2.rpl dumped from a Wii U.

**Comment by EatYOuRNuGGeT** [comment]
i have a wii u with tiramisu. Do you know where i can find these files?  im not too tech savvy but maybe i could find them if you point me in the right direction.

**Comment by Squall-Leonhart** [comment]
They are part of the Firmware libraries, 

storage_slc/sys/title/00050010/1000400a/code

**Comment by EatYOuRNuGGeT** [comment]
Hello, i put the correct files into the CafeLibs folder and enabled Shared Libraries in the game profile and while sound has definitely improved the game still suffers from that awfully sounding crackling static issue. SFX quality has improved though. 
Here's my log in case you'll want to take a peak at it. 
[log.txt](https://github.com/cemu-project/Cemu/files/14169419/log.txt)

**Comment by Squall-Leonhart** [comment]
what is the cubeb buffer length on your end?

**Comment by EatYOuRNuGGeT** [comment]
im not exactly sure what that is so i quickly looked up what you mean and came upon this reddit post https://www.reddit.com/r/cemu/comments/la3tdk/anyone_elses_game_sound_like_this/
even though this clip is from BOTW this is what my Hyrule Warriors sounds like and changing API or latency in Cemu doesn't make a difference. if you're talking about something in my audio drivers then i wouldn't know i just use a wired headset on my xbox controller. all other games sound perfect though.  Here's a video of Hyrule warriors from someone that has the same audio issue i have  https://www.youtube.com/watch?v=8VfeLjAUp8U  i'd be happy to tell you my "cubeb buffer lenght" if you could tell me where i find this info.

**Comment by Squall-Leonhart** [comment]
are you sending surround audio to the xbox audio device, its stereo only.

**Comment by EatYOuRNuGGeT** [comment]
No i am using stereo.

**Comment by Squall-Leonhart** [comment]
![image](https://github.com/cemu-project/Cemu/assets/1098176/8975867b-522d-4c62-b40e-ffed20629604)

I thought it was stereo, its actually mono (left channel only)

**Comment by Squall-Leonhart** [comment]
Somethings whacky with my receiver and controller, keeps alternating between mono and stereo with audio crackles mixed in even on the test.

**Comment by EatYOuRNuGGeT** [comment]
Have you tried to boot up Hyrule warriors on your cemu by chance? do you have the same audio problem? im not sure if i can troubleshoot this issue with settings alone. maybe its just a bug with cemu that needs to be patched?

**Comment by Squall-Leonhart** [comment]
I have, the issue doesn't occur on HDMI or Realtek audio, and some quick looking into such issues with the xbox 3.5mm jack indicates its a common problem with it.

**Comment by EatYOuRNuGGeT** [comment]
alright then, i'll test if i can get it working with a different audio setup in the future. thanks.

**Comment by JetnReadyBoogy** [comment]
Hey mates, I was wondering if their was any breakthrough with this?
